### ISSUE TITLE
Accessibility: Exhibit details button needs accessible label

### ISSUE THREAD

**Issue by wrenaria** [UCDavisLibrary/ucdlib-wp-plugins#201]
Links are supposed to have unique text / provide context for screen readers. The featured exhibit buttons all use the same `Exhibit Details` text so if a screenreader reads all the links on the page, it will just read the same text 3 times.

Can we make that block include an aria label or something to pull the exhibit title, for example `aria-label="Exhibit details: Exhibit Title"`?

_Related: [SiteImprove Report for Online Exhibits](https://my2.siteimprove.com/Inspector/728315/A11Y/Page?pageId=247463709567&impmd=130BA75F00FDF9C4E13726DF81FAB862&expand=potential&issue=failed+cantTell&wcag=two&decision=none#/sia-r81/cantTell)_

**Comment by spelkey-ucd** [comment]
Released [v4.5.0](https://github.com/UCDavisLibrary/ucdlib-wp-plugins/releases/tag/v4.5.0)
